src/GeometryEditor/EditorToolbarController.cs

```diff
@@ -78,6 +77,7 @@ private void RaiseActiveINPC()
         public bool IsReshapeActive => IsLineInputActive && lineInputMode == "Reshape";
         public bool IsCutActive => IsLineInputActive && lineInputMode == "Cut";
         public bool IsLineInputActive => GeometryEditor == lineInputEditor;
+        public bool IsEditingActive => GeometryEditor is not null;
```

Comment:
I don't think this is correct. There's also a property off of this whether it has started.

Reply:
As GeometryEditor is nullable property, It still needs check before IsStarted Property. So I am thinking of this change.

```suggestion
        public bool IsEditingActive => GeometryEditor is not null  && GeometryEditor.IsStarted;
```